node/service/src/service.rs

```diff
@@ -136,14 +140,38 @@ pub fn new_partial(
 		telemetry
 	});
 
-	// See https://github.com/paritytech/polkadot-sdk/pull/4639 for how to enable the fork-aware pool.
+	// default to Fork-Aware pool
+	// Warning: This overrides all the TransactionPoolParams with the default values (config.transaction_pool)
+	let pool_params = TransactionPoolParams {
```

Comment:
@wilwade This is one way of doing it but I don't like it since we have no other choice than use default params due to Rust limits.